dashboard/test/models/script_test.rb

```diff
@@ -1593,35 +1593,35 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal true, script_copy.hidden
   end
 
-  test 'clone script with inactive variant' do
-    script_file = File.join(self.class.fixture_path, "test-fixture-variants.script")
+  test 'clone script with variants' do
+    script_file = File.join(self.class.fixture_path, "test-fixture-experiments.script")
```

Comment:
why use test-fixture-experiments instead of test-fixture-variants here?

Reply:
Ah, sorry for not explaining this. the ScriptsControllerTest makes sure we never use the script edit gui when editing a script what has script level variant experiments. This is the test that was failing: https://github.com/code-dot-org/code-dot-org/blob/4035d5dd27ea2bd93a05c18cfe54082966a838ae/dashboard/test/controllers/scripts_controller_test.rb#L780-L779

so, I switched around which script we are using to test the new behavior so that the test above and the one that follows it would both still pass.